pages/index.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <div class="relative">
         <div class="flex flex-col h-full bg-dark">
-            <Header class="mb-10 || md:mb-20"/>
-            <Hero class="mb-10 || lg:mb-14 || 2xl:mb-20"/>
-            <Steps class="z-20"/>
+            <Header class="mb-10 || md:mb-20" />
+            <Hero class="mb-10 || lg:mb-14 || 2xl:mb-20" />
+            <Steps class="z-20" />
```

Comment:
Why are `||` used?

Reply:
Already answered! Just serve the better readability.